http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GceInterpreter.scala

```diff
@@ -70,6 +62,18 @@ class GceInterpreter[F[_]: Parallel: ContextShift: Logger](
                                                     config.gceConfig.zoneName,
                                                     params.runtimeConfig.machineType)
 
+      // Determine is user persisent disk has been formatted before
+      isGceFormatted <- params.runtimeConfig match {
```

Comment:
I was thinking return `isGceFomatted` as part of the block below...since we're already doing this check in https://github.com/DataBiosphere/leonardo/pull/1457/files#diff-44f5fd773b99b0ea53e82a7b38b3d8e2R140

Reply:
Think this is needed here in order to pass to `RuntimeTemplateValuesConfig.fromCreateRuntimeParams`?

Reply:
yea it will...just moving things around a bit....so `RuntimeTemplateValuesConfig.fromCreateRuntimeParams` will be moved down as well...

Reply:
I was thinking to keep as is since we do all the templating first (defining `resourceConstraints`, `templateParams`, and `templateValues`) then build the disks and instance. if i moved it, i would also need to move the templating to happen after we build the disk but before we build the instance (since the instance `templateValues` when building it) and it seemed better to keep the logic flow that way